gcp-deployer/machine_setup_configs.yaml

```diff
@@ -106,11 +111,28 @@ items:
       - ${PRIVATEIP}
       EOF
 
```

Comment:
Using metadata is fine in the short run. However, it is not secure to have the ca on something as widely accessible as the vm metadata. What is the longterm plan?

Reply:
Unfortunately I am not confident yet in what the long term plan is / should be. I do agree that this is not a good long term solution, I can create an issue tracking this with a link back to this PR and details on why the current approach is not the best for security.

Reply:
Sure. Also add a link to the issue on this thread as well. :)

Reply:
Created issue: https://github.com/kubernetes-sigs/cluster-api/issues/162.